src/harness/values.yaml

```diff
@@ -42,7 +42,7 @@ global:
   saml:
     autoaccept: false
   license:
-    cg: ''
```

Comment:
these would require multiple changes inside sub-charts of helm-platform where global.cg usages were already there.
cc: @kapilgarg1996

Reply:
@pratmit is it something being changed everywhere across harness ?

Reply:
Yes, CG is renamed to FG across Harness.